string_array.go

```diff
@@ -1,11 +1,30 @@
 package main
 
 import (
+	"flag"
 	"strings"
 )
 
+var _ flag.Getter = StringArrayGetter{}
+
 type StringArray []string
 
+type StringArrayGetter struct {
+	stringArray *StringArray
```

Comment:
pointer necessary? A slice is already a reference.

Reply:
I see why this is done here. But it's super complex for a simple mutable data structure. Do we even need StringArray ? Can't we just have a simple `[]string` field in this struct and normal mutating `Set` changing the field?

Reply:
Or differently: why not just add `Get()` to StringArray itself?

Reply:
> Or differently: why not just add Get() to StringArray itself?

because the interface requires the receiver not to be a pointer

> Can't we just have a simple []string field in this struct and normal mutating Set changing the field?

I _think_ this did not work

Reply:
... but I can make it a `*[]string`